examples/deferred_processing/README.md

```diff
@@ -0,0 +1,340 @@
+    f9c620d1a4cc421a9c3923b749e9e6ed edges_dp           21 minutes ago     3 seconds 0       1 + 0 / 1 57.27KiB 0B       success 
+    ```
+
+11. If we want to trigger another job in `montage_dp`,
+    based on what's currently in the `dev` branch,
+    we simply update `master` to point at `dev`.
+    
+    ```sh
+    $ pachctl create branch edges_dp@master --head dev
+    $ pachctl list job
+    ID                               PIPELINE           STARTED            DURATION  RESTART PROGRESS  DL       UL       STATE 
+    4b05f2c0ddf14ab0a4ca8f4ee2d6d9ff montage_dp         About a minute ago 3 seconds 0       1 + 0 / 1 693.8KiB 673.6KiB success 
+    1a8d2fefc2ba450e9102ffd6b74e503d montage_dp         3 minutes ago      4 seconds 0       1 + 0 / 1 693.8KiB 673.6KiB success 
+    c5feebe5c6ae48e0adc4f5bd7be9a9ac edges_dp           3 minutes ago      3 seconds 0       1 + 3 / 4 175.1KiB 0B       success 
+    7d6aa9bbe1484e2e9ac60a7f84147e1b montage_dp         24 minutes ago     1 second  0       0 + 0 / 0 0B       0B       success 
+    f7a7d71dd7e74a3f9c48d4b9d2897ba3 edges_dp           24 minutes ago     4 seconds 0       2 + 1 / 3 181.1KiB 0B       success 
+    f9c620d1a4cc421a9c3923b749e9e6ed edges_dp           24 minutes ago     3 seconds 0       1 + 0 / 1 57.27KiB 0B       success 
+    ```
+
+12. We won't cause another job to trigger on `montage_dp` if we commit more data to `images_dp_1`.
+    It will only trigger a job in `edges_dp`.
+
+    ```sh
+    $ pachctl put file images_dp_1@master:2GI70mb.jpg -f http://imgur.com/2GI70mb.jpg
+    $ pachctl list job
+    ID                               PIPELINE           STARTED            DURATION  RESTART PROGRESS  DL       UL       STATE   
+    96439afdaf8249be81773c89fc87b366 edges_dp           31 seconds ago     3 seconds 0       1 + 4 / 5 204KiB   0B       success 
+    4b05f2c0ddf14ab0a4ca8f4ee2d6d9ff montage_dp         About a minute ago 3 seconds 0       1 + 0 / 1 693.8KiB 673.6KiB success 
+    1a8d2fefc2ba450e9102ffd6b74e503d montage_dp         3 minutes ago      4 seconds 0       1 + 0 / 1 693.8KiB 673.6KiB success 
+    c5feebe5c6ae48e0adc4f5bd7be9a9ac edges_dp           3 minutes ago      3 seconds 0       1 + 3 / 4 175.1KiB 0B       success 
+    7d6aa9bbe1484e2e9ac60a7f84147e1b montage_dp         24 minutes ago     1 second  0       0 + 0 / 0 0B       0B       success 
+    f7a7d71dd7e74a3f9c48d4b9d2897ba3 edges_dp           24 minutes ago     4 seconds 0       2 + 1 / 3 181.1KiB 0B       success 
+    f9c620d1a4cc421a9c3923b749e9e6ed edges_dp           24 minutes ago     3 seconds 0       1 + 0 / 1 57.27KiB 0B       success 
+    ```
+
+13. We can create another "stable" branch called `stable_1_1`.
+    
+    ```sh
+    $ pachctl create branch edges_dp@stable_1_1 --head master
+    ```
+
+14. We can move master to our old stable branch, `stable_1_0`,
```

Comment:
once stable part removed -- this step can be removed completely IMO. We want to just focus on deferred processing and dont need to reintroduce ideas like skipped datums